src/MiNET/MiNET/Net/MCPE Protocol Documentation.md

```diff
@@ -1297,6 +1311,9 @@ Wiki: [Crafting Data](https://github.com/NiclasOlofsson/MiNET/wiki//Protocol-Cra
 | Name | Type | Size |
 |:-----|:-----|:-----|
 |Recipes | Recipes |  |
+|Some Array Size | UnsignedVarInt |  |
+|Some Array Size 2 | UnsignedVarInt |  |
```

Comment:
These are potion recipes. There are "potion mix" recipes (e.g. awkward -> something else) and "container mix" recipes (e.g. normal potion -> splash potion). https://github.com/Sandertv/gophertunnel/blob/7ecfeb4cf6edfd49ca0a106378f89cd62de14a0a/minecraft/protocol/recipe.go#L12-L72